src/view/com/modals/ProfilePreview.tsx

```diff
@@ -27,12 +27,12 @@ export function Component({did}: {did: string}) {
     data: profile,
     error: profileError,
     refetch: refetchProfile,
-    isFetching: isFetchingProfile,
+    isLoading: isLoadingProfile,
   } = useProfileQuery({
     did: did,
   })
 
-  if (isFetchingProfile || !moderationOpts) {
+  if (isLoadingProfile || !moderationOpts) {
     return (
```

Comment:
we'd rather use `isLoading` instead of `isFetching` since we want to show the avatar/banner/bio as long as it's been loaded once. the 15 second stale time will handle updating it if it has changed.